### DISCUSSION TITLE
webserver hit with this error "Error: Unknown option '--open'"

### DISCUSSION THREAD

**Discussion by arunvenkat1911** [webpack/webpack#17304] [Q&A]
Hi, 

I am using these following things on my system. 

NODE: 16.3.0
NPM: 7.15.1

When I try to run this command: 
set NODE_OPTIONS=--max_old_space_size=8192 && webpack serve --open --config ../webpack.config.js --progress

I am getting this error: 
[webpack-cli] Error: Unknown option '--open'

Please help.

Arun.

**Comment by alexander-akait** [comment]
Please run `npm ls webpack-dev-server`
